suites/portal/studyViewerTest.js

```diff
@@ -1,46 +1,53 @@
-Feature('Study Viewer / Requestor');
+Feature('Study Viewer');
 
-const chai = require('chai');
-const { interactive, ifInteractive } = require('../../utils/interactive.js');
+const studyViewerTasks = require('../../services/portal/studyViewer/studyViewerTasks.js');
+const studyViewerProps = require('../../services/portal/studyViewer/studyViewerProps.js');
 
-const { expect } = chai;
+/* Setup after the test */
+After(async (home) => {
+  home.complete.logout();
+  // TBC - delete the access for the mainAcct user from arborist and requestor db
+});
 
-/* User without access */
-Scenario('User1 has no access to download @manual', ifInteractive(
-  async () => {
-    const result = interactive(`
-              1. Go to the Study Viewer Page
-              2. Select the dataset that is needed for research
-              3. Click on 'Request Access' button to acquire access to download
-              4. User will receive request_id from 'Request Access Queue'
-              5. User3 (who has Requestor access) makes a manual call to Requestor to validate the 'request_id' received
-              6. Go back to the page and user will have access to 'Download' button 
-              7. Click in 'Download' button to download the file from indexd
-          `);
-    expect(result.didPass, result.details).to.be.true;
-  },
-));
+// User does not log in and has no access. User see 'Login to Request access' button
+Scenario('User logs in and requests the access @studyViewer', async (I, home, users, login) => {
+  home.do.goToHomepage();
+  login.complete.login(users.mainAcct);
+  studyViewerTasks.goToStudyPage();
+  await studyViewerTasks.clickRequestAccess();
+  // request id from requestor db
+  await studyViewerTasks.getRequestId();
+  await studyViewerTasks.putRequest();
+  I.refreshPage();
+  I.wait(5);
+  // await studyViewerTasks.clickDownload();
+  await studyViewerTasks.deleteRequest();
+});
 
-Scenario('User2 has access to download @manual', ifInteractive(
-  async () => {
-    const result = interactive(`
-              1. Go to the Study Viewer Page
-              2. Select the dataset that is needed for research
-              3. Click on 'Download' button to download the file from indexd
-          `);
-    expect(result.didPass, result.details).to.be.true;
-  },
-));
+// User doesnot login and want to request access
+Scenario('User doesnot login and requests the access', async (I, users, login) => {
```

Comment:
@atharvar28 , please utilize the new CodeceptJS 3 syntax here.
It's super easy, just replace this:
```
async (I, users, login)
```
with this:
```
async ({ I, users, login })
```

I can push commits to your branch if you want me to do it.
Just let me know.